modules/ephemeris/src/main/scala/gem/horizons/EphemerisParser.scala

```diff
@@ -87,7 +96,8 @@ object EphemerisParser {
     */
   def parsedElements[F[_]]: Pipe[F, String, ParseResult[Ephemeris.Element]] =
     _.through(fs2.text.lines)
-     .dropThrough(_.trim =!= SOE)
+     .dropWhile(_.trim =!= SOE)
+     .drop(1)
```

Comment:
This *should* be equivalent but I believe there is an [FS2 bug](https://github.com/functional-streams-for-scala/fs2/issues/934) here.